components/authz-service/storage/postgres/migration/migration.go

```diff
@@ -36,7 +36,7 @@ func (c *Config) Migrate(dataMigConf datamigration.Config) error {
 	migrationsTable := ""
 	ctx := context.Background()
 
-	l.Infof("Running db migrations from %q", migrationsPath)
+	l.Info("Running db migrations...")
```

Comment:
This migration path was already output in the log from the caller so don't need to duplicate it here. That lets me use my notion of making this an "open bracket" as explained above. The "close bracket" is down at the end of the function.

Reply:
While we're here, maybe "Initializing migrations" makes more sense. 🤷‍♂